### PR TITLE
ENH: viz tweaks

### PR DESCRIPTION
**Pull request by dengemann**
bring back ica component numbers in plot_sources_evoked

screenshot 2015-11-30 18 00 40

cc @jaeilepp @Eric89GXL

### CODE REVIEW

File: tutorials/plot_ica_from_raw.py [hunk 1]
```diff
@@ -57,7 +57,7 @@
 ecg_epochs = create_ecg_epochs(raw, tmin=-.5, tmax=.5, picks=picks)
 
 ecg_inds, scores = ica.find_bads_ecg(ecg_epochs, method='ctps')
-ica.plot_scores(scores, exclude=ecg_inds, title=title % 'ecg', label='ecg')
+ica.plot_scores(scores, exclude=ecg_inds, title=title % 'ecg', labels='ecg')
```
Comment (Eric89GXL) [inline tutorials/plot_ica_from_raw.py]: So this was not working before, then?

Comment (dengemann) [inline tutorials/plot_ica_from_raw.py]: yes

File: mne/viz/ica.py [hunk 2]
```diff
@@ -267,9 +269,13 @@ def _plot_ica_sources_evoked(evoked, picks, exclude, title, show, labels=None):
 
     for exc_label, ii in zip(exclude_labels, picks):
         if exc_label is not None:
-            if labels is not None:
-                exc_label = exc_label.split(' - ')[1]
-            color = label_colors[exc_label]
+            # create look up for color ...
+            if ' - ' in exc_label:
```
Comment (Eric89GXL) [inline mne/viz/ica.py]: So we don't need the `labels is not None` check anymore, right? It looks like we don't based on how it's constructed above, just want to double-check.

File: tutorials/plot_ica_from_raw.py (hunk 1 above)
Comment (agramfort) [inline tutorials/plot_ica_from_raw.py]: not an API change?

Comment (dengemann) [inline tutorials/plot_ica_from_raw.py]: it did not exist before

Comment (dengemann) [inline tutorials/plot_ica_from_raw.py]: I've added this very recently

File: mne/viz/ica.py (hunk 2 above)
Comment (dengemann) [inline mne/viz/ica.py]: we don't need this second check any more, you also see when unwrapping a bit that this block follows an is None check. You see it?

**Comment by dengemann** [comment]
ping can we merge?

**Comment by agramfort** [comment]
+1

**Pull request merged**